docs/Users_Guide/systemconfiguration.rst

```diff
@@ -1454,6 +1454,56 @@ fcst dictionary item. If this is the case, then use these variables. If
 it is not set, the values in the corresponding
 FCST_VAR<n>_[NAME/LEVELS/THRESH/OPTIONS] will be used in the ens dictionary.
 
+Probabilistic Forecast Fields
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If processing probabilistic forecast fields, there are additional configuration
+variables that are used to properly format the field info that is passed into
+the wrapped MET configuration files.
+:term:`FCST_IS_PROB` is used to process probabilistic data::
+
+    [config]
+    FCST_IS_PROB = True
+    FCST_VAR1_NAME = APCP
```

Comment:
Wondering if we should use a more likely example, like verifying the ensemble frequencies from the NetCDF output of Ensemble-Stat or Gen-Ens-Prod?
```suggestion
    FCST_VAR1_NAME = APCP_24_A24_ENS_FREQ_gt0.0
```